src/components/ChatLog.js

```diff
@@ -0,0 +1,47 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import {DateTime} from 'luxon';
+
+
+const ChatLog = (props) => {
+    const chatComponents = entries => (entries.map(entry => {
+        return (
+            <li key={entry.id}>
+                <ChatEntry
+                id={entry.id}
+                sender={entry.sender}
+                body={entry.body}
+                timeStamp={entry.timeStamp}
+                liked={entry.liked}
+                updateHeart={props.updateHeart}
```

Comment:
[style nit] I prefer the props to be indented by one when they're split across lines like this.